sdk/verifiers/python/tests/test_receipt_v2.py

```
@@ -0,0 +1,93 @@
+
+
+def test_receipt_cli_json(capsys) -> None:  # type: ignore[no-untyped-def]
+    code = main(["receipt", str(VALID_SPANNED_V2), "--key", V2_GOLDEN_PUBLIC_KEY, "--json"])
+    captured = capsys.readouterr()
+    assert code == 0
+    body = json.loads(captured.out)
+    assert body["valid"] is True
+
+
+def test_receipt_cli_requires_key(capsys) -> None:  # type: ignore[no-untyped-def]
+	code = main(["receipt", str(VALID_SPANNED_V2), "--json"])
```

Ruff (0.15.15)

[warning] Indentation contains tabs

(W191)